sysutils/pfSense-pkg-System_Patches/files/usr/local/www/system_patches_edit.php

```diff
@@ -71,8 +71,8 @@
 
 	/* input validation */
 	if (empty($_POST['location'])) {
-		$reqdfields = explode(" ", "patch");
-		$reqdfieldsn = array(gettext("Patch Contents"));
+		$reqdfields = explode(" ", "descr patch");
```

Comment:
I don't think description should be required. It wasn't in the past, and there are likely many patch entries already present without it. There is no compelling reason to force it.